p2p/src/net/default_backend/peer.rs

```diff
@@ -201,16 +201,19 @@ where
             }
         }
 
+        let mut was_accepted = false;
```

Comment:
This variable can probably be of the `SetFlag` type.

Reply:
Sure, I will update it next time, thanks.